src/views/migration/components/MigrationToolPhrase.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Trans } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import ExternalLink from '@kubevirt-utils/components/ExternalLink/ExternalLink';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import {
+  MIGRATION_TOOL_DOCUMENTATION_URL,
+  PackageManifestKind,
+  SubscriptionsKind,
+} from '../constant';
+
+type MigrationToolPhraseProps = {
+  operator: PackageManifestKind;
+  subscription: SubscriptionsKind;
+};
+
+const MigrationToolPhrase: React.FC<MigrationToolPhraseProps> = ({ operator, subscription }) => {
+  const { t } = useKubevirtTranslation();
+
+  if (!operator)
+    return (
+      <Alert
+        variant={AlertVariant.danger}
```

Comment:
it's perfectly fine to not install MTV on this cluster, you can still use it if it's installed on a different cluster.